resources/help/prompt.md

```diff
@@ -14,6 +14,25 @@ Returns the line currently typed into the prompt
 
 ***prompt.set(input)***
 Sets the line typed in the prompt. Replacing any current data.
+This set the cursor position to the end of the new prompt.
+
+##
+
+***prompt.get_cursor_pos() -> number***
+Gets the cursor position in the input prompt
+
+- Returns the current index of the cursor starting at 0
```

Comment:
I think it might be nice to have this API return a 1-indexed value since most users will immediately do that conversion themselves to work with the prompt data as a Lua string. In the prompt mask API I converted to/from the Lua style indexing in the UserData impl that sits at the edge of the Rust/Lua code.

Reply:
Mmm, I see your point :+1:

Reply:
I have no strong opinions on this. Let's go with the Lua format here.